aioftp/server.py

```diff
@@ -1042,6 +1059,12 @@ def __init__(self,
 
                     connection.passive_server.close()
 
+                    if self.available_data_ports is not None:
+
+                        self.available_data_ports.put_nowait(
+                            (0, connection.passive_server_port)
+                        )
+
```

Comment:
If we have no passive connection for session, but have `available_data_ports`, then we will put (0, 0) to queue.